src/test/tests.ts

```diff
@@ -234,6 +241,31 @@ describe('SubscriptionManager', function() {
     });
   });
 
+  it('can use an async filter function', function(done) {
+    const query = `subscription Filter2($filterBoolean: Boolean){
+       testFilter(filterBoolean: $filterBoolean)
+      }`;
+    const callback = function(err, payload){
```

Comment:
As the test case is structured an err would be unexpected but easy enough to check.